### ISSUE TITLE
[APP SUBMITTED]: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas)

### ISSUE THREAD

**Issue by doncoque** [SickChill/SickChill#5512]
### INFO
Python Version: **2.7.15 (default, Aug 16 2018, 14:17:09) [GCC 6.4.0]**
Operating System: **Linux-5.0.0-15-generic-x86_64-with**
Locale: None
Branch: **master**
Commit: SickChill/SickChill@86207e9e728f4685e5b92a14674d2ae4e6e1279b
Link to Log: https://gist.github.com/5c7a85dd062e4a9924d59f470538e896
### ERROR
```
SEARCHQUEUE-BACKLOG-73696 :: [Torrent9] :: [86207e9] Request failed: Cloudflare captcha challenge presented for ww1.torrent9.nz (cfscrape cannot solve captchas) (<type 'str'>)
```
---
_STAFF NOTIFIED_: @SickChill/owners @SickChill/moderators

**Comment by pro-src** [comment]
Hi @doncoque,

I've created an issue at https://github.com/Anorov/cloudflare-scrape/issues/259
Please follow that issue.

Thanks.

**Comment by pro-src** [comment]
Fixed by #5516 and is on the master branch.
